fog/ledger/enclave/api/src/lib.rs

```diff
@@ -98,6 +101,62 @@ pub trait LedgerEnclave: ReportableEnclave {
 
     /// Add a key image data to the oram Using thrm -rf targete key image
     fn add_key_image_data(&self, records: Vec<KeyImageData>) -> Result<()>;
+
+    // LEDGER ROUTER / STORE SYSTEM
+
+    /// Begin a connection to a Fog Ledger Store. The enclave calling this
+    /// method, most likely a router, will act as a client to the Fog Ledger
+    /// Store.
+    fn ledger_store_init(&self, ledger_store_id: ResponderId) -> Result<NonceAuthRequest>;
+
+    /// As a "Store" in a Router/Store system,
```

Comment:
nit: This is a bit confusing. Consider changing to something like: 
`  /// Accept a connection to a Fog Ledger Router instance acting as a frontend
    /// to the Fog Ledger Store.
`

Reply:
Addressed in [278d8c1](https://github.com/mobilecoinfoundation/mobilecoin/pull/2896/commits/278d8c1f98696a964ebd09ae87c4b2d5e316cfd0)